### ISSUE TITLE
Erlport 1.0 release?

### ISSUE THREAD

**Issue by scohen** [hdima/erlport#33]
The website says 1.0 alpha is out, but that's an old version. We've been using erlport pretty happily in production for over a year, and I think you should release a new version.

Also, the rubygems and pypi packages should be updated. Pypi is at 0.6, which is very old.

**Comment by filmor** [comment]
Yes, please. Even if it's not 1.0, the currently "newest release" is nearly 3 years old and is missing a lot of bugfixes.

**Comment by ghost** [comment]
It looks like this library isn't maintained anymore. Has everybody moved to Thrift/Riffed?

**Comment by scohen** [comment]
@amarandon Our use case for erlport is different than Thrift, but if you want an RPC layer, Pinterest has just released [elixir thrift](https://github.com/pinterest/elixir-thrift), which blows the doors off of Riffed in terms of ease of use and speed.
